UM/Qt/qml/UM/MenuItem.qml

```diff
@@ -0,0 +1,61 @@
+import QtQuick 2.12
+import QtQuick.Layouts 1.12
+import QtQuick.Controls 2.12
+
+MenuItem
+{
+    id: root
+
+    property alias shortcut: _shortcut.sequence
+    property bool indicatorVisible: root.icon.source.length > 0 || root.checkable
+
+    Shortcut
+    {
+        id: _shortcut
+        enabled: root.enabled
+        onActivated: root.triggered()
+        sequence: root.action!= null ? root.action.shortcut: null
+    }
+
+    function replaceText(txt)
+    {
+        var index = txt.indexOf("&")
+        if(index >= 0)
+        {
+            txt = txt.replace(txt.substr(index, 2), ("<u>" + txt.substr(index + 1, 1) +"</u>"))
+        }
+        return txt
+    }
+
+    contentItem: RowLayout
+    {
+        spacing: 0
+        width: root.width
+        opacity: root.enabled ? 1 : 0.5
+
+        Item
+        {
+            // Spacer
+            width: root.indicatorVisible ? root.indicator.width + 4 : 0
```

Comment:
Shouldn't we pick margin size from the theme.json here?

Reply:
Whooops :$ You're right! Fixed in 66ebf83030a635e1e56bbcae936cfc9880c26a02